test/utils/run_tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+mod_dir=$( cd "$( dirname "$0" )"/../.. && pwd )/node_modules
+mocha="$mod_dir"/mocha/bin/mocha
+istanbul="$mod_dir"/istanbul/lib/cli.js
+
+runTest ( ) {
+    if [ "$1" = "sqlite" ]; then
+        echo "Running with SQLite backend"
+        export RB_TEST_BACKEND=sqlite
+        rm -f restbase
+    else
+        echo "Running with Cassandra backend"
+        export RB_TEST_BACKEND=cassandra
+        sh ./test/utils/cleandb.sh
+    fi
+
+    if [ "$2" = "test" ]; then
+        "${mocha}"
+    elif [ "$2" = "coverage" ]; then
+        "${istanbul}" cover node_modules/.bin/_mocha -- -R spec
+    fi
+}
+
+if [ "x$2" == "x" ]; then
```

Comment:
Ups, that should have been

```
if [ "x$2" = "x" ]; then
```

I got carried away by the *bashism* in my earlier comment.